image_generator.py

```diff
@@ -0,0 +1,33 @@
+import cv2
+
+
+# returns an tuple with channel values as respectice indices at the specified image pixel
+def get_pixel_channel_values(image, x, y):
```

Comment:
I'm not sure we need these functions, you should be able to implement all of these as one liners using numpy slicing or numpy reshape.

Reply:
CV color detection and CV ball detection won't be needing this file, so we will be removing this file from our spec.